Makefile

```diff
@@ -31,6 +31,15 @@ clean-venv:
 test: install
 	uv run --locked python -m unittest discover tests
 
+.PHONY: lint
+lint: install
+	uv run --locked ruff check --fix
```

Comment:
I think this will also fail fast. Maybe add `-`?

Reply:
Aha! Thank you.

Reply:
https://github.com/tpgillam/mt2/pull/80/commits/bc4e390e5408384c49cbc41768c784856f48fcb3